samples/deployment/aws/README.md

```diff
@@ -0,0 +1,48 @@
+# Setup Guide
+
+## Setup dependencies
+1. Create Python virtual environment
+    ```bash
+    virtualenv -p python3.6 ./venv
+    source venv/bin/activate
+    pip install -r requirements.txt
+    ```
+
+1. Download dynamic inventory for AWS 
+    ```bash
+    # Get Dynamic inventory
+    wget https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/ec2.py
+    wget https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/ec2.ini    
+    chmod +x ec2.py
+    ```
+
+1. Set environment variables
+    ```bash
+    export AWS_ACCESS_KEY_ID='... aws access key ...' 
+    export AWS_SECRET_ACCESS_KEY='... aws secret key ...'
+    export AWS_REGION='... aws region ...'
```

Comment:
Let's move it to `environment` block of a playbook.

Reply:
credentials can be defined in `ec2.ini` file however even there it's not recommended 

> Supplying AWS credentials here is not recommended, as it introduces
> non-trivial security concerns. When going down this route, please make sure
> to set access permissions for this file correctly, e.g. handle it the same
> way as you would a private SSH key.